packages/gatsby-theme-patternfly-org/helpers/codeTitleTransformer.js

```diff
@@ -0,0 +1,43 @@
+const visit = require('unist-util-visit');
+
+function exampleCaptions() {
```

Comment:
Basic `h3` support as a rehype plugin. This should be a remark plugin but `gatsby-theme-mdx` nukes changes we make to the MDAST, so it has to be a rehype plugin.